udata_front/theme/gouvfr/datagouv-components/src/components/utils/Placeholder.vue

```diff
@@ -0,0 +1,37 @@
+<!--
+-
+name: Placeholder
+category: 5 - Interactions
+-
+# Placeholder
+A very simple component that takes a `type` (dataset, reuse, post...) and an optional `src`. If `src` is defined, it simply displays the image (and all props passed are passed to the image).
```

Comment:
This comment can be moved to the description of stories about the placeholder component